src/library.js

```diff
@@ -448,6 +448,9 @@ LibraryManager.library = {
   // TODO: There are currently two abort() functions that get imported to asm module scope: the built-in runtime function abort(),
   // and this function _abort(). Remove one of these, importing two functions for the same purpose is wasteful.
   abort__sig: 'v',
+#if !MINIMAL_RUNTIME
+  abort__proxy: 'sync',
+#endif
```

Comment:
I think it's still the right thing to proxy to the main thread in the minimal runtime, even if what runs in `_abort()` is different?

Reply:
@juj specifically suggested not doing that.

Reply:
I guess it doesn't make much difference, wither way MINIMAL_RUNTIME just throws here.. and doesn't do stuff like shut down all the other threads.

Reply:
The comment there is about not having `onAbort`, which is true. But still, it seems like a good idea to abort on the main thread if we abort at all? Anyhow, how about adding a TODO comment for minimal runtime, if we aren't sure.

Reply:
OK I agree we might as well proxy it for MINIMAL_RUNTIME too.   The only reason not too would be if it increases code size for all threaded programs.